### PR TITLE
modify cameratest Operation

### PR DESCRIPTION
**Pull request by songmiao**

### CODE REVIEW

File: cocos/platform/android/java/project.properties [hunk 1]
```diff
@@ -12,4 +12,4 @@
 
 android.library=true
 # Project target.
-target=android-10
+target=android-19
```
Comment (super626) [inline cocos/platform/android/java/project.properties]: reset this change after build on android platform

**Comment by liuliang2004** [comment]
Modify the code specification

File: tests/cpp-tests/proj.android/project.properties [hunk 2]
```diff
@@ -8,6 +8,6 @@
 # project structure.
 
 # Project target.
-target=android-10
+target=android-19
```
Comment (super626) [inline tests/cpp-tests/proj.android/project.properties]: do not change this

File: tools/cocos2d-console [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit 1a71fe7be45db504f3bdc0e24e8d7eca83f2fc1a
+Subproject commit b49fad3fb2dbc396e71a5b1178cf2532309cd174
```
Comment (super626) [inline tools/cocos2d-console:1]: should not change this. update submodule first.

File: tests/cpp-tests/Classes/Camera3DTest/Camera3DTest.cpp [hunk 4]
```diff
@@ -539,12 +624,130 @@ void Camera3DTestDemo::updateCamera(float fDelta)
...
             }
         }
+        if(_bZoomOut == true)
+        {
+            if(_camera&& _cameraType!=CameraType::FirstCamera)
+            {
+                Vec3 cameraPos = _camera->getPosition3D();
+                if(cameraPos.y <= 200)
+                {
+                    cameraPos += cameraPos.getNormalized();
```
Comment (super626) [inline tests/cpp-tests/Classes/Camera3DTest/Camera3DTest.cpp]: Why use this?
Does not  camera move with the same speed?
Zoom in means move toward target, and zoom out move to opposite direction.

File: tests/cpp-tests/Classes/Camera3DTest/Camera3DTest.cpp [hunk 5]
```diff
@@ -539,12 +624,130 @@ void Camera3DTestDemo::updateCamera(float fDelta)
...
+            }
+        }
+        if(_bZoomIn == true)
+        {
+            if(_camera&& _cameraType!=CameraType::FirstCamera)
+            {
+                Vec3 cameraPos = _camera->getPosition3D();
+                if(cameraPos.y >= 50)
+                {
+                    cameraPos -= cameraPos.getNormalized();
```
Comment (super626) [inline tests/cpp-tests/Classes/Camera3DTest/Camera3DTest.cpp]: same as above

File: tests/cpp-tests/Classes/Camera3DTest/Camera3DTest.cpp [hunk 6]
```diff
@@ -262,29 +279,85 @@ void Camera3DTestDemo::onEnter()
...
+    //auto menuItem4 = MenuItemLabel::create(label4, CC_CALLBACK_1(Camera3DTestDemo::rotateCameraCallback,this,-10));
+    //_decRot = menuItem4;
     auto label5 = Label::createWithTTF(ttfConfig,"free ");
     auto menuItem5 = MenuItemLabel::create(label5, CC_CALLBACK_1(Camera3DTestDemo::SwitchViewCallback,this,CameraType::FreeCamera));
     auto label6 = Label::createWithTTF(ttfConfig,"third person");
     auto menuItem6 = MenuItemLabel::create(label6, CC_CALLBACK_1(Camera3DTestDemo::SwitchViewCallback,this,CameraType::ThirdCamera));
     auto label7 = Label::createWithTTF(ttfConfig,"first person");
     auto menuItem7 = MenuItemLabel::create(label7, CC_CALLBACK_1(Camera3DTestDemo::SwitchViewCallback,this,CameraType::FirstCamera));
-    auto menu = Menu::create(menuItem1, menuItem2, menuItem3, menuItem4, menuItem5, menuItem6, menuItem7, nullptr);
+    auto menu = Menu::create(menuItem5, menuItem6, menuItem7, nullptr);
```
Comment (super626) [inline tests/cpp-tests/Classes/Camera3DTest/Camera3DTest.cpp]: remove unused code.
variables start with menuItem5?

**Comment by songmiao** [comment]
remove unused code and modify zoom limit

**Pull request closed without merging**